server/controllers/postEvent.js

```diff
@@ -0,0 +1,58 @@
+const {
+  Event, Coffee, Lunch, Equipment,
+} = require('../database/models');
+
+exports.postEvent = (req, res) => {
+  const { userId } = req;
+  const {
+    title,
+    start_date,
+    end_date,
+    org_name,
+    price,
+    capacity,
+    note,
+    coffee_note,
+    coffee_price,
+    coffee_time,
+    lunch_note,
+    lunch_price,
+    lunch_time,
+    equibment_price,
+    equipment_note,
+  } = req.body;
+
+  return Event.create({
```

Comment:
You don't have to use ```return``` here, the data will return by default inside ```then()``` because sequlize is a promise

Reply:
done